CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Getting Started
+
+- View the [README](https://github.com/KamandPrompt/baat-cheet/blob/master/README.md) to get your development environment up and running. 
+- Learn how to [format pull requests](#submitting-a-pull-request).
+- Read how to [rebase/merge upstream branches](#configuring-remotes).
+- Follow our [code of conduct](https://github.com/KamandPrompt/baat-cheet/blob/master/CODE_OF_CONDUCT.md).
+- [Find an issue to work on](https://github.com/KamandPrompt/baat-cheet/issues) and start smashing!
+
+# Contributing Guidelines [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/KamandPrompt/baat-cheet/issues)
+
+When contributing to this repository, please first discuss the change you wish to make via an issue.
+
+Make sure you read the [README.md](https://github.com/KamandPrompt/baat-cheet/blob/master/README.md).
+
+Do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. If you have got any questions, please email to us at [pc@iitmandi.ac.in](mailto:pc@iitmandi.ac.in).
```

Comment:
Instead of the email here, we should put in the link to [Zulip stream](https://kamandprompt.zulipchat.com/#narrow/stream/201590-Baat-Cheet.20Project) . (Even though as of now its only allowed for college students to signup).

Reply:
Okay, I will add that.

Reply:
Done @abhigyank ! 👍